nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/test/java/org/apache/nifi/processors/azure/storage/ITPutAzureBlobStorage_v12.java

```diff
@@ -57,6 +66,14 @@ public void testPutBlobWithSimpleName() throws Exception {
         assertSuccess(getContainerName(), BLOB_NAME, BLOB_DATA);
     }
 
+    @Test
+    public void testPutBlobApplyBlobMetadata() throws Exception {
+        runProcessor(BLOB_DATA);
+
+        assertSuccess(getContainerName(), BLOB_NAME, BLOB_DATA);
+        assert ((ITProcessor) runner.getProcessor()).blobMetadataApplied;
```

Comment:
The Java `assert` keyword should not be used in tests, instead, JUnit `assertTrue` should be used.

Reply:
Thanks – fixed in 2d80fa23f2bf1310d6bbaff5baf8ba66fd6d9946.